### ISSUE TITLE
[Bug]: The simulated data click is initiated again, and the date type is displayed incorrectly

### ISSUE THREAD

**Issue by Huangxiao-mas** [oceanbase/odc#3987]
### ODC version

ODC433

### OB version

independent

### What happened?

The simulated data click is initiated again, and the date type is displayed incorrectly
![image](https://github.com/user-attachments/assets/54763ef3-cabb-45d8-bb94-b3bde177d1fd)

### What did you expect to happen?

。

### How can we reproduce it (as minimally and precisely as possible)?

```
CREATE TABLE "GL_PROD_BAL_HIST" (
  "SYSTEM" VARCHAR2(20) NOT NULL,
  "BRANCH" VARCHAR2(20) NOT NULL,
  "CCY" VARCHAR2(3) NOT NULL,
  "GL_CODE" VARCHAR2(20) NOT NULL,
  "PROFIT_CENTRE" VARCHAR2(12) NOT NULL,
  "PROD_TYPE" VARCHAR2(50) NOT NULL,
  "BUSI_PROD" VARCHAR2(50) NOT NULL,
  "MARKETING_PROD" VARCHAR2(50) NOT NULL,
  "ACTUAL_BAL" NUMBER(17,2),
  "DR_ACTUAL_BAL" NUMBER(17,2),
  "CR_ACTUAL_BAL" NUMBER(17,2),
  "RUN_DATE" DATE NOT NULL,
  "COMPANY" VARCHAR2(20) NOT NULL,
  "TRAN_TIMESTAMP" TIMESTAMP(6),
  "CREATE_TIMESTAMP" TIMESTAMP(6) DEFAULT sysdate,
  CONSTRAINT "PK_GL_PROD_BAL_HIST" PRIMARY KEY ("SYSTEM", "BRANCH", "CCY", "GL_CODE", "PROFIT_CENTRE", "PROD_TYPE", "BUSI_PROD", "MARKETING_PROD", "COMPANY", "RUN_DATE")
) COMPRESS FOR ARCHIVE REPLICA_NUM = 1 BLOCK_SIZE = 16384 USE_BLOOM_FILTER = FALSE TABLET_SIZE = 134217728 PCTFREE = 0
 partition by range(RUN_DATE)
(partition P_202308 values less than (TO_DATE('2023-09-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202309 values less than (TO_DATE('2023-10-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202310 values less than (TO_DATE('2023-11-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202311 values less than (TO_DATE('2023-12-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202312 values less than (TO_DATE('2024-01-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202401 values less than (TO_DATE('2024-02-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202402 values less than (TO_DATE('2024-03-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202403 values less than (TO_DATE('2024-04-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202404 values less than (TO_DATE('2024-05-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202405 values less than (TO_DATE('2024-06-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202406 values less than (TO_DATE('2024-07-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202407 values less than (TO_DATE('2024-08-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202408 values less than (TO_DATE('2024-09-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202409 values less than (TO_DATE('2024-10-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202410 values less than (TO_DATE('2024-11-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202411 values less than (TO_DATE('2024-12-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202412 values less than (TO_DATE('2025-01-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202501 values less than (TO_DATE('2025-02-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202502 values less than (TO_DATE('2025-03-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202503 values less than (TO_DATE('2025-04-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202504 values less than (TO_DATE('2025-05-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202505 values less than (TO_DATE('2025-06-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202506 values less than (TO_DATE('2025-07-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202507 values less than (TO_DATE('2025-08-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202508 values less than (TO_DATE('2025-09-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202509 values less than (TO_DATE('2025-10-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202510 values less than (TO_DATE('2025-11-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202511 values less than (TO_DATE('2025-12-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202512 values less than (TO_DATE('2026-01-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202601 values less than (TO_DATE('2026-02-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202602 values less than (TO_DATE('2026-03-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202603 values less than (TO_DATE('2026-04-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')),
partition P_202604 values less than (TO_DATE('2026-05-01 00:00:00', 'SYYYY-MM-DD HH24:MI:SS', 'NLS_CALENDAR=GREGORIAN')))

```

**Comment by HSunboy** [comment]
Fixed In Commit [cde598c983709bbebf6b34952101d749d03336aa](https://github.com/oceanbase/odc-client/commit/cde598c983709bbebf6b34952101d749d03336aa)
            ------
            艺泽
            Wed Dec 18 17:36:39 2024 +0800

**Comment by sl01388797** [comment]
![Image](https://github.com/user-attachments/assets/63cf5015-fe83-49bc-9738-9b8cfa6fbcdc)
pass
